packages/core/config/core-config-server-internal/src/index.ts

```diff
@@ -8,4 +8,5 @@
 
 export { coreDeprecationProvider } from './deprecation';
 export { ensureValidConfiguration } from './ensure_valid_configuration';
+// eslint-disable-next-line @kbn/imports/no_boundary_crossing
```

Comment:
@spalger how would we resolve this?

Reply:
The problem here is that `test_utils` are assumed to be code only intended for use by tests, but it's imported from the `index.ts` file of the package, meaning that it's included in the distributable and these test helpers are exposed to everyone who imports this file. If these helpers are actually used in other packages then we will need to move them to a new `test-helpers` package (though we can't define the type of the package yet, that would be it's purpose and it's name should reflect that somehow). Then we can import the `@kbn/core-config-server-test-helpers` package in the local tests and in the other packages which use them.

If these test helpers are not actually needed outside this package then this `export ... from` statement can just be removed.